lib/azure/AzureDataStore.ts

```diff
@@ -159,15 +157,20 @@ export default class AzureDataStore {
    * @param key
    * @param callback
    */
-  public has (key: any, callback: any): void {
-    this.blobService.doesBlobExist(this.container, this.getFullKey(key), (err, result, _response) => {
-      if (err) {
-        callback(err, false);
-      } else if (result && result.exists) {
-        callback(null, true);
-      } else {
-        callback(null, false);
-      }
+  public async has (key: any): Promise<boolean> {
+    return new Promise((resolve, reject) => {
+      this.blobService.doesBlobExist(this.container, this.getFullKey(key), (err, result, _response) => {
+        if (err) {
+          reject(err);
+          return;
+        } else if (result && result.exists) {
+          resolve(true);
+          return;
+        } else {
+          resolve(false);
+          return;
+        }
+      });
```

Comment:
More of a nit comment (no changes required), `return` doesn't look needed? "Just in case"?

Reply:
Not needed, I just put return after every resolve/reject because it can fall through in some cases. In this case, it doesn't matter because we are in if else and at the end of a function.